tests/conftest.py

```diff
@@ -56,6 +57,17 @@ def unvouched_user(stored_users):
     return stored_users['unvouched']
 
 
+@pytest.fixture(scope='session')
+def github_non_nda_user(stored_users):
+    return stored_users['github_non_nda']
```

Comment:
I'm concerned about having fake GitHub users stored in our credentials. Have we spoken to GitHub about the possibilities of test accounts?

Reply:
I contacted the github support, will do updates when I hear from them.